docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+* Built-in data validation for requests and responses
+
+## Terminology
+
+**Data validation** automatically validates the user input and the response of your AWS Lambda function against a set of constraints defined by you.
+
+**Event handler** is a Powertools for AWS feature that processes an event, runs data parsing and validation, routes the request to a specific function, and returns a response to the caller in the proper format.
+
+**[OpenAPI schema](https://www.openapis.org/){target="_blank"}** is an industry standard JSON-serialized string that represents the structure and parameters of your API.
+
+**Action group** is a collection of two resources where you define the actions that the agent should carry out: an OpenAPI schema to define the APIs that the agent can invoke to carry out its tasks, and a Lambda function to execute those actions.
+
+**Large Language Models (LLM)** are very large deep learning models that are pre-trained on vast amounts of data, capable of extracting meanings from a sequence of text and understanding the relationship between words and phrases on it.
+
+**Agent for Amazon Bedrock** is an AWS service to build and deploy conversational agents that can interact with your customers using Large Language Models (LLM) and AWS Lambda functions.
+
+## Getting started
+
+???+ tip "All examples shared in this documentation are available within the [project repository](https://github.com/aws-powertools/powertools-lambda-python/tree/develop/examples)"
+
+### Install
+
+!!! info "This is unnecessary if you're installing Powertools for AWS Lambda (Python) via [Lambda Layer/SAR](../../index.md#lambda-layer){target="_blank"}."
+
+You need to add `pydantic` as a dependency in your preferred tool _e.g., requirements.txt, pyproject.toml_. At this time, we support both Pydantic V1 and V2. For a future major version, we will only support Pydantic V2.
+
+### Required resources
+
+To build Agents for Amazon Bedrock, you will need:
+
+| Requirement                                                                                                           | Description                                                                    | SAM Supported | CDK Supported |
+|-----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------|:-------------:|:-------------:|
+| [Lambda Function](#your-first-agent)                                                                                  | Defines your business logic for the action group                               |       ✅       |       ✅       |
+| [OpenAPI Schema](#generating-openapi-schemas)                                                                         | API description, structure, and action group parameters                        |       ❌       |       ✅       |
```

Comment:
I thought openapi schemas are compatible with SAM if you use `AWS::Serverless::Api.DefinitionBody` or AWS::Serverless::HttpApi.DefinitionBody` (depending on OpenAPI version) to specify it

Reply:
This is not about defining the OpenAPI schema of an API Gateway, the OpenAPI Schema is a configuration parameter of the Agent for Amazon Bedrock.